Add list of Z-Wave Command Classes supported in Home Assistant (#34698)
home-assistant/home-assistant.github.io: source/_integrations/zwave_js.markdown

### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)

A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.

> ## Z-Wave Command Classes supported in Home Assistant

Reviewer: ```suggestion
## Z-Wave Command Classes supported in Home Assistant when queried
```

> ## Z-Wave Command Classes supported in Home Assistant when queried

Author: ```suggestion
## Z-Wave Command Classes Home Assistant responds to when queried
```
or maybe like this?

Reviewer: Yeah, that's good.

> The following table lists Command Classes that Home Assistant will respond to when queried by other devices, together with the implemented version and required security class:

Author: ```suggestion
The following table lists the Command Classes together with the implemented version and required security class. These are the Command Classes that Home Assistant will respond to when queried by other devices. 
```

> | Command Class                 | Version | Security Class  |

Author: ```suggestion
| Command Class                 | Version        | Security Class  |
```

> | Command Class                 | Version        | Security Class  |

Author: ```suggestion
| Command Class                 | Version         | Security Class  |
```

> | Command Class                 | Version         | Security Class  |

Author: ```suggestion
| Command Class                 | Version          | Security Class  |
```

### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)

A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.

## Z-Wave Command Classes supported in Home Assistant

> The following table lists Command Classes that are supported in Home Assistant, together with their version and their required
> Security Class.

Reviewer 2: This is tricky...

In Z-Wave terms, a **supported** command class (CC) means that another Z-Wave device can send HA controlling commands of the same CC, and will get a correct response. Those have to be documented.

However users likely understand something else under **supported**, namely which CCs they can interact with in their devices. That list is much, much longer, does not have to be documented (and IMO should not).

I think we need to distinguish between those meanings here. I'm not 100% sure how though. Maybe
 ```suggestion
The following table lists Command Classes that Home Assistant will respond to when queried by other devices, together with the implemented version and required security class:
```

> | Command Class                 | Version Required | Security Class  |

Reviewer 2: ```suggestion
| Command Class                 | Version | Security Class  |
```
